package.json

```diff
@@ -20,6 +20,7 @@
     "deploy-storybook-pr": "yarn relay && NODE_ENV=production build-storybook -s ./public -o storybook_build ",
     "emit-types": "tsc --declaration --emitDeclarationOnly --listEmittedFiles --jsx react --outDir dist",
     "lint": "tslint -c tslint.json --project tsconfig.json",
+    "omg": "om g",
```

Comment:
just a thought -- maybe `omk` instead?

Reply:
Seems a bit more memorable, keeps a strong connection with omakase and kinda implies the main command. `omg` is emoji-speak 😄

Reply:
Are you suggesting to rename the main command? I.e. resulting in `omk g …` ?

Reply:
Yup yup. (Om also brings to mind a few other projects)

Reply:
Ok, in that case I'm going to leave this PR as-is for now, because until we might rename it I think it's reasonable to think people might misspell it.

Reply:
👍